### PR TITLE
[MRG] manylinux1 wheels and travis build improvements

### PR DESCRIPTION
**Pull request by luizirber**
This PR configures Travis to build wheels for manylinux1 (the catch-all version for compatible binary Python wheels). Wheel building is executed only on tags, and for now it is deploying to GitHub Releases (we want to deploy it to PyPI too)

I'm using an OAuth token linked to my account to upload the wheels to GitHub, so we might want to figure out another approach (maybe make a `diblab-release` account, and add it to PyPI/bio.tools and other services too?)

Bonus: since I generated the khmer wheels for this, I also uploaded them to PyPI and now the Travis jobs run way faster (because we don't need to recompile khmer every time...)

Not covered here:
- We are currently not builiding on macOS, so next step is to turn it on again on Travis
- Windows (via appveyor) seems achievable, but need to fix khmer first...
- [Build stages](https://docs.travis-ci.com/user/build-stages/) would also be a welcome improvement, but it requires more thinking/rewriting on the travis configuration file

## checklist

- [x] Is it mergeable?
- [x] `make test` Did it pass the tests?
- [x] `make coverage` Is the new code covered?
- [x] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [x] Was a spellchecker run on the source code and documentation after
  changes were made?

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,46 +1,48 @@
...
+- "/snap/bin/ipfs daemon --init --offline &"
+- tox -e $TOX_ENV
+before_deploy:
+- git clean -xfd
+- docker run --rm -v `pwd`:/io quay.io/pypa/manylinux1_x86_64 $PRE_CMD /io/.travis/build-wheels.sh
+  $TOX_ENV
+deploy:
+  provider: releases
+  api_key:
+    secure: CTO7kiWKLper/nJU093pp796P6MhV2IPz4M2Ws0//YOqU2gKZalyGmAdJtqsFk0+eZthnOp0GQzuSJYpN0+jOx3IU+SInnMg7HdLooyfLxgjDri9xOMBtTiCzSbY77NnLKNk4Nj8idb3Gfjp0F+/5TIRlpInTZ7iYGoG1ZYM/kCyBiTkzDd8tO1LBKhWBad3h3bv099OcmoYT7GETuS9LMmUYJLOe/aotOfOhSttNxYIIIY2RmjFQjWBZCAWPx9P8PnTdhDmrMQL9kRBz+z9/E+ikaaMgtx9QTsF75mZn1qix0+a0LwvRaXKCzYCZ7KGUaSs/npIrBbtXxxA7wK4GFTGb2ERpJOzp4MGzwLiToQlNOHZ48F2N3RgQVL0v+rfvGjt14csWRA21jfrvP2yk4G8lrHZsVxdBeMyPw8/4EnFm307zZbd8LY21byz2K9450mzExLGTx+0AWKQGyygT6PPOXYsNqxC5XEMNZDKTknaGQ1RjiZwRnZAq9quk4KNNmfbgRnuEVKcK0SXeudq3wRwbyA5jT/cYaScfBAdhAgge6tto9aEss+mUEEhm1ZmCxSAfTRKJgE3+ug4npTSSb0KLdGWW4V+ygQ2t+kA9G+1j7U01CGG3jY7aPc/mnJF7Sb2XP0+PNS9eLZpXa6rlJ/1ieMSqppuvvgQgHmKHVs=
```
Comment (ctb) [inline .travis.yml]: are you sure you want this checked in?

Comment (luizirber) [reply .travis.yml]: Yup:
https://github.com/travis-ci/travis.rb#encrypt
https://docs.travis-ci.com/user/deployment/releases/

**Comment by luizirber** [comment]
(for those curious: this is how many tries it took to figure out this PR
https://travis-ci.com/luizirber/sourmash-1/builds
https://github.com/luizirber/sourmash-1/releases
)

**Comment by luizirber** [comment]
Ready for review and merge @ctb

**Pull request merged**